app/controllers/api/greetings_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Api
+  class GreetingsController < ApplicationController
+    def index
+      render json: { message: 'Hello World' }
+    end
```

Comment:
- [ ] You have done great creating the API endpoint for getting your message. However, according to the project requirements, you are required to **get random messages** and now the message being returned by your endpoint is only one "Hello world" which means that you are not meeting the project requirements in any way.

- To ensure that you are getting a random message (since you have seed data :clap: ) You could get all the data and then get one at a time. This could be done this way:

```
def index
      message = Greeting.all.sample
      render json: message 
    end
```